src/MakeListingDisplay/components/InputField.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+/**Functional Component that creates an input field. Takes in a fieldName and fieldType as props
+to fill in the name and type attributes of the input element, respectively*/
+function InputField(props) {
+    return (
+        <div>
+            <p>{props.fieldHeader}</p>
+            <input name={props.fieldName} type={props.fieldType} required/>
+        </div>
+    );
+}
```

Comment:
Have you considered the approach outlined in React's form guide where the <input> fields are used directly in the render() call of MakeListingDisplay? The reason being there's some fields that an input tag can have (required, min, etc.) that aren't represented here in your current InputField model and it might be easier to use the input tag directly versus having to define an InputField that defines an <input>

https://reactjs.org/docs/forms.html

Reply:
Because each input has a header, name, type, and is required, building a component that automatically does these things and also allows us to differentiate if need be (min, step, etc.) gives us more freedom while also keeping things clean